### ISSUE TITLE
Nexus 6P cannot take pictures

### ISSUE THREAD

**Issue by Coder206** [mozilla/webmaker-android#2465]
I am trying to take a picture to add to Webmaker on my Nexus 6P and the camera never shows up, no menus, just keeps showing my project.

**Comment by Coder206** [comment]
No, this is not an issue. I did not enable camera and storage in my permissions. Could we have a popup to remind users that they must allow camera and storage to use the camera?

**Comment by ryanwarsaw** [comment]
Hey there! Thanks for filing an issue, runtime permission support (and Android 6.0+ support in general) is currently being tracked over at mozilla/webmaker-core#685. PRs are welcome if you would like to submit one, otherwise I can put some time aside next week and look into it further.

**Comment by Coder206** [comment]
@ryanwse I am interested in helping out here. It's just a bad time for me. It may take me some time to get it fixed but sign me up!
